### ISSUE TITLE
Simultaneous predictions from different models not possible

### ISSUE THREAD

**Issue by uzfm** [SciSharp/TensorFlow.NET#905]
```
        static SciSharp.Models.IImageClassificationTask[] task = new SciSharp.Models.IImageClassificationTask[2];
        static tensorflow tf = new tensorflow();
        static SciSharp.Models.ModelWizard  wizard = new SciSharp.Models.ModelWizard();
        static string _assetsPath = Path.Combine(TFdt.DT.PathAssets, TFdt.DT.PathDataSet);
        static string [] Model_Pach = { Path.Combine(_assetsPath, "Model_Camera 1.pb"), Path.Combine(_assetsPath, "Model_Camera 2.pb") };
        static public string[] LablePach = { Path.Combine(_assetsPath, "Labels_Camera 1.txt"), Path.Combine(_assetsPath, "Labels_Camera 2.txt") };

 static public ModelPredictResult Predict(Image<Bgr, byte> img_bgr ,int ID) {
      
            Mat output = new Mat();
            CvInvoke.CvtColor(img_bgr, output, Emgu.CV.CvEnum.ColorConversion.Bgr2Rgb);
            var output1 = output.ToImage<Rgb, byte>().Resize(300, 300, Emgu.CV.CvEnum.Inter.Linear);
            output1.ToUMat().ConvertTo(output, Emgu.CV.CvEnum.DepthType.Cv32F, 1.0f / 255.0f);
            //NDArray imgR = np.array(output.GetData(), TF_DataType.TF_FLOAT);
            Tensor tensorr = tf.convert_to_tensor(output.GetData());
            var img_final = tf.expand_dims(tensorr, 0);
            //predict image
            if (task[ID] == null)
            {
                task[ID]= wizard.AddImageClassificationTask<TransferLearning>(new SciSharp.Models.TaskOptions
                {
                    ModelPath = Model_Pach[ID],
                    LabelPath = LablePach[ID]
           

                }); ;
            }
        
              var result = task[ID].Predict(img_final);
            return result;
        }
```

**Tensorflow.RuntimeError: 'Current graph is not default graph. Default Graph Key: graph-2/, Current Graph Key: graph-1/'**

**Comment by CrimyTheBold** [comment]
Have you tried the latest version (0.70.0) ? There was some significant changes to support multi-threading (with one model per thread)

**Comment by uzfm** [comment]
![ghb](https://user-images.githubusercontent.com/25766808/149500925-3e9b4889-e5c1-4fbc-84a6-2f8fac942b37.png)

Everything works great with one model. If I add another model for the next forecast I get an error

**Comment by uzfm** [comment]
I use
**SciSharp.TensorFlow.Redist-Windows-GPU 2.7.0
SciSharp.Models.ImageClassification 0.4.0
TensorFlowSharp 1.15.1**

**Comment by uzfm** [comment]
I can download a small test program

**Comment by uzfm** [comment]
[((https://drive.google.com/file/d/1R57DewCRe66w_CTnHerWuvk76ZKdxkgV/view?usp=sharing))]

**Comment by Oceania2018** [comment]
@uzfm I downloaded your sample project, it works well, can't reproduce from my side.
![image](https://user-images.githubusercontent.com/1705364/149629161-728ffbcc-c6a9-4b16-9f1b-f6d435398284.png)

**Comment by uzfm** [comment]
Prediction is good only for the first time. If we make  prediction next time we will get error!

**Comment by uzfm** [comment]
> @uzfm I downloaded your sample project, it works well, can't reproduce from my side. ![image](https://user-images.githubusercontent.com/1705364/149629161-728ffbcc-c6a9-4b16-9f1b-f6d435398284.png)

Prediction is good only for the first time. If we make  prediction next time we will get error!

**Comment by uzfm** [comment]
![Image1](https://user-images.githubusercontent.com/25766808/149657887-9b49d20f-6efa-43f5-9517-cee3965615e5.png)
Next time the prediction gets an error.
 If you make a forecast for one model, everything is fine.

**Comment by Oceania2018** [comment]
@uzfm This issue has been fixed, Just upgrade to ImageClassification v0.4.2 and TensorFlow.NET v0.70.1
